docs/src/main/asciidoc/logging.adoc

```diff
@@ -77,6 +77,32 @@ class SimpleBean {
 
 NOTE: The logger instances are cached internally. Therefore, a logger injected e.g. into a `@RequestScoped` bean is shared for all bean instances to avoid possible performance penalty associated with logger instantiation.
 
+=== Logging with Panache
```

Comment:
The title might need to change now that this feature lives in core?

Reply:
I don't really care, but I also don't really know. My understanding is that the "Panache" term (as much as I dislike it) is the Quarkus "brand" for "hey look how Quarkus can simplify your life with a little bit of magic", in which case, "Logging with Panache" seems like a perfect fit?

Reply:
Now I am confused, isn't Panache related to ORM specifically?

Reply:
They way I understand it, this is a widespread misconception :-) @FroMage any comments on that? Thanks!

Reply:
Yeah @Ladicek is right. It's supposed to be a general brand for our opinionated magic.

Reply:
I'm afraid our users may be confused with this as well, because we don´t have anything (at least in our documentation) about Panache that is not related to ORM.

Reply:
Well, we have MongoDB with Panache.

Reply:
> Well, we have MongoDB with Panache.

Still ORM no? :) Anyway, I am fine with it as long as the message is not confusing to our users

Reply:
I'm actually with @gastaldi on this, it is confusing, plus I never really liked the term :rofl:, so... as I said, I don't really care, but I also don't really know.

Reply:
It's supposed to be generic and related to magic, so not related to ORM, which is why we're always careful to not abbreviate `ORM with Panache` to just `Panache`.